### PR TITLE
doc: add @H4ad as member

### PR DESCRIPTION
**Pull request by H4ad**
Fixes https://github.com/nodejs/performance/issues/86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ For the current list of team members, see the project
 * [@tniessen](https://github.com/tniessen) - Tobias Nießen
 * [@trevnorris](https://github.com/trevnorris) - Trevor Norris
 * [@UlisesGascon](https://github.com/UlisesGascon) - Ulises Gascón
+* [@H4ad](https://github.com/H4ad) - Vinícius Lourenço
```
Comment (UlisesGascon) [inline README.md:36]: I will suggest to follow the alphabetical order (nickname)

**Review by UlisesGascon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (H4ad) [reply README.md]: Oh, I was so excited that I forgot to sort by the nickname instead real name hahahuah

**Review by anonrig — APPROVED** [review]

**Comment by anonrig** [comment]
cc @nodejs/performance

**Review by RafaelGSS — APPROVED** [review]
❤️

**Comment by RafaelGSS** [comment]
@anonrig wanna me to invite him?

**Comment by anonrig** [comment]
> @anonrig wanna me to invite him?

Appreciate it, if you can. Thanks @RafaelGSS!

**Comment by RafaelGSS** [comment]
Done. @H4ad you should have received the invite in your email.

**Comment by anonrig** [comment]
Welcome @H4ad!

**Pull request merged**